### ISSUE TITLE
UX: how do I add an item to my collection?

### ISSUE THREAD

**Issue by paololombardi** [Cores-ts/Fundingbox-support#37]
After creating my first collection, I entered by clicking "My collection" in my profile
Problem: I struggled to add my first item. 
Solution: It would be good to have a button "Add item" somewhere in the collection page.

Then I tried to add a New Content, and I added a link to NESTA's article on accelerators.
Problem: there is no tag for such a content type
Solution: add tag "News" or "Info" or "Report" or similar

![screenshot 2016-02-03 12 24 10](https://cloud.githubusercontent.com/assets/16993463/12781043/3e9b6b82-ca72-11e5-8df9-2b79dd1a0a6b.png)

Finally I looked for my new content to add it to my collection: I could not find it. I used the scroll buttons to go through the "Recently added" and it was not there.

Result: UX = confusion (and I still do not know how to add an item)

**Comment by xurxo** [comment]
Right now an article it's not the type of content we planned to show as a "profile".
Our initial intention is all the news, videos, articles... should be shared inside a community/space in "Spaces", then we could show the most shared, relevant articles in the platform... but we probably need to discuss this.

To go to your curated items please go to "User menu"->"my profile"->"Curated"
Or directly http://www.fundingbox.com/users/@paololombardi/curated

To save an item to a collection, just click the "Save" or "Save this" button you'll find on each profile or profile card.

I don't see the option to add an item directly from a collection because it is as easy as "[Search](http://fundingbox.com/search/nesta) and Save", but we'll add some suggestions there

![captura de pantalla 2016-02-03 12 44 33](https://cloud.githubusercontent.com/assets/715608/12781624/cccbe78a-ca75-11e5-8bd8-17b460fc9b76.png)
![captura de pantalla 2016-02-03 12 44 43](https://cloud.githubusercontent.com/assets/715608/12781627/ce2f2998-ca75-11e5-8501-3512f9592d17.png)

**Comment by paololombardi** [comment]
Suggestion:
An "Add item" from the collection page could point to a selected search containing both your curated list of items, and items with keywords related to the collection (and then scrolling capabilities).
So it could point to a 'search results' standard page containing those, and you can then click "Save" as usual, but without going back to the homepage and start a search there.

**Comment by xurxo** [comment]
Yes, it seems an option, I'll mark this add this as a nice to have improvement.
